benchmarks/attention/benchmark_attention.py

```diff
@@ -9,6 +9,8 @@
 import torch
 import nvtx
 import transformer_engine
+
+sys.path.append("../../")
```

Comment:
Hmm, even though this is a standalone benchmark, this isn't ideal

Reply:
Removed. There were some changes I made when debugging stuff.